### PR TITLE
News entry about Credential Manager integration

### PR DESCRIPTION
**Pull request by Vinnl**
## Summary

image

## Full entry

image

### CODE REVIEW

File: en/app.ftl [hunk 1]
```diff
@@ -150,6 +150,13 @@ whatsnew-feature-offer-countdown-snippet = Get { -brand-name-relay-premium } bef
 whatsnew-feature-offer-countdown-description = Get { -brand-name-relay-premium } before { $end_date } and enjoy unlimited masking at our intro month-to-month price.
 whatsnew-feature-offer-countdown-cta = Upgrade now
 
+whatsnew-feature-firefox-integration-heading = { -brand-name-relay } is now available in { -brand-name-firefox } password manager
+# A preview of the full content of `whatsnew-feature-firefox-integration-description`.
+# When translating, please make sure the resulting string is of roughly similar
```
Comment (peiying2) [inline en/app.ftl:155]: Suggestion to the comment: `Try to keep this string short, not longer than the English version.`

Comment (Vinnl) [reply en/app.ftl]: Thanks, that's more concise. I applied the same change to the other comments with the same content: 3237718

**Comment by Vinnl** [comment]
Thanks for the review! This won't go live in a while, so I'll leave this open for now so it can be merged at the same time as potential other new strings.

**Review by peiying2 — APPROVED** [review]
Thanks for making the sweeping changes in this file. LGTM.

**Pull request merged**